clusterloader2/api/validation/validation.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"fmt"
+
+	"k8s.io/perf-tests/clusterloader2/api"
+)
+
+// VerifyConfig does simple validatio for config and all included structures.
+// Returns error is any condition is not satisfied.
+func VerifyConfig(c *api.Config) error {
+	if c.AutomanagedNamespaces < 1 {
+		return fmt.Errorf("Expected one or more namespaces!")
+	}
+	for _, tuningSet := range c.TuningSets {
+		if err := verifyTuningSet(&tuningSet); err != nil {
+			return err
+		}
+	}
+	for _, step := range c.Steps {
+		if err := verifyStep(&step); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// verifyStep requires to either Phases or Measurements be non-empty,
+// but never both at the same time.
+func verifyStep(s *api.Step) error {
+	for _, phase := range s.Phases {
+		if err := verifyPhase(&phase); err != nil {
+			return err
+		}
+	}
+	for _, measurement := range s.Measurements {
+		if err := verifyMeasurement(&measurement); err != nil {
+			return err
+		}
+	}
+	if (len(s.Phases) == 0) == (len(s.Measurements) == 0) {
+		return fmt.Errorf("Empty Phases and Measurements!")
```

Comment:
errors should start with small letter (everywhere here)

Reply:
done